stable/kong/templates/deployment.yaml

```diff
@@ -252,4 +263,12 @@ spec:
         - name: custom-nginx-template-volume
           configMap:
             name: {{ .Release.Name }}-kong-default-custom-server-blocks
-
+{{- if (and (not .Values.ingressController.enabled) (eq .Values.env.database "off")) }}
+        - name: kong-custom-dbless-config-volume
+          configMap:
+            {{- if .Values.dblessConfig.configMap }}
+            name: {{ .Values.dblessConfig.configMap }}
+            {{- else }}
+            name: {{ .Release.Name }}-kong-custom-dbless-config
```

Comment:
Please add a `define` to `_helpers.tpl` and use that here and in the ConfigMap name value as well to avoid typos.

Reply:
If added a define for the configMap name just like what has been done for cassandra/postgress.